app/styles/components/speaker-list.scss

```diff
@@ -13,3 +13,20 @@
 .word-break {
   overflow-wrap: break-word;
 }
+
+.thumbnail {
+  position: relative;
+  width: 220px;
```

Comment:
Why two classes

Reply:
to not use image a backgroung. 
one is for cropping , and other is for using with img tag

Reply:
other wise code will be like 
`<div class = "thumbnail" style = "background-image: url(" image url ")/> `

Reply:
I didn't get what you are saying.

Reply:
https://jonathannicol.com/blog/2014/06/16/centre-crop-thumbnails-with-css/

Reply:
check link mentioned by @iamareebjamal

Reply:
@kushthedude the solution is from same link ..https://stackoverflow.com/questions/11552380/how-to-automatically-crop-and-center-an-image
scroll down and you will see it. it has 74 upvotes.

Reply:
> https://jonathannicol.com/blog/2014/06/16/centre-crop-thumbnails-with-css/

this was a reference for you to understand what i implemented.

Reply:
@maze-runnar 
```css
.center-cropped {
  object-fit: none; /* Do not scale the image */
  object-position: center; /* Center the image within the element */
  height: 300px;
  width: 300px;
}
```
Try this at the place of two classes

Reply:
its not cropping as supposed to perhaps , 
![Screenshot from 2020-01-22 23-59-48](https://user-images.githubusercontent.com/56407566/72922614-597b2f00-3d73-11ea-9af2-8956c4638f29.png)

Reply:
but i will make changes if you say so .

Reply:
@kushthedude please confirm , i should make changes here or not ?